docker/images/noetic/Dockerfile

```diff
@@ -22,6 +22,26 @@ RUN mkdir -p /tmp/ws/src \
   && find ./ -name "COLCON_IGNORE" | xargs cp --parents -t /tmp/ws/src \
   || true
 
+FROM ros:noetic-ros-base-focal AS timem-builder
+
+RUN apt-get update \
+  && apt-get install --no-install-recommends -y \
+    git \
```

Comment:
nit: same.

Reply:
I'll keep it that way because there's a lot of copying and pasting from Dockerfiles and this format makes it very clear which packages are being installed.